### PR TITLE
Tj2 14/doughnut chart

### PR DESCRIPTION
**Pull request by ZenVega**
This PR is for the a doughnut chart. The design is here: https://www.figma.com/file/s12w00ODssXTtPQfSizlGj/TSB-Sachen?node-id=678%3A121523

The custom tooltip and legend are unused but still in the repo as possible options

### CODE REVIEW

File: src/components/PieChart/Tooltip/Tooltip.module.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@import '../../../styles/base.scss';
+
+:local {
```
Comment (vogelino) [inline src/components/PieChart/Tooltip/Tooltip.module.scss]: I would remove deadcode. Git does version it anyway. :)

File: src/components/PieChart/index.jsx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        index === elementIndex ? color : `${color}40`
+      )
+    );
+  };
+
+  const onChartHover = (event, chartElement) => {
+    if (chartElement[0]) {
+      // const index = chartElement[0].index;
+      event.native.target.style.cursor = "pointer";
+      //   if (index != tooltipData.index) {
```
Comment (vogelino) [inline src/components/PieChart/index.jsx]: I'm against keeping dead code around. I would prefer removing it.

File: src/components/PieChart/index.jsx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+          <div
+            className={cx(cn.infoBox, cn.clickIcon)}
+            onClick={onClickIconClick}
+          >
+            <ClickIcon />
+          </div>
+        )}
+      </div>
+      <Overview data={overview} />
+      {/* <Legend /> */}
```
Comment (vogelino) [inline src/components/PieChart/index.jsx]: Same hier

**Review by vogelino — COMMENTED** [review]
Good job! I have some comments:

I noticed that the price in the doughnut is often unreadable or broken down to two lines:
Screenshot 2022-06-07 at 15 28 00

Once a segment is clicked, you cannot go back to the state where all slices are visible. It could be possible to do the same interaction on hover on desktop and click on mobile. And on second click to set the state back to the initial one.

Also, clicking on the inner ring always selects "Senatsverwaltung für Wirtschaft, Energie und Betriebe", even when the currently highlighted slice is another one.

**Comment by ZenVega** [comment]
@vogelino went over it and fixed your comments.

**Comment by vogelino** [comment]
Great changes! Thx for taking them into account! Let's merge this! :)

**Pull request merged**